### PR TITLE
Correction of Soares. Addition of Nieuwstadt

### PR DESCRIPTION
**Pull request by ilopezgp**
Soares (SCAMPy) corresponds to Soares_moist in PyCLES. 

Nieuwstadt (SCAMPy) is Soares in PyCLES.

### CODE REVIEW

File: Cases.pyx [hunk 1]
```diff
@@ -123,15 +126,110 @@ cdef class Soares(CasesBase):
         return
 
     cpdef initialize_surface(self, Grid Gr, ReferenceState Ref ):
-        self.Sur.zrough = 1.0e-4
+        self.Sur.zrough = 0.16 #1.0e-4 0.16 is the value specified in the Nieuwstadt paper.
```
Comment (yairchn) [inline Cases.pyx]: so both Soares case and Nieuwstadt use this value ?

File: Cases.pyx [hunk 2]
```diff
@@ -123,15 +126,110 @@ cdef class Soares(CasesBase):
...
+
+    cpdef update_surface(self, GridMeanVariables GMV, TimeStepping TS):
+        self.Sur.update(GMV)
+        return
+    cpdef update_forcing(self, GridMeanVariables GMV, TimeStepping TS):
+        self.Fo.update(GMV)
+        return
+
+cdef class Nieuwstadt(CasesBase):
+# Case based on (Nieuwstadt et al, 1993): LES of the CBL: A comparison of 4 codes
```
Comment (yairchn) [inline Cases.pyx]: maybe just the full reference ? 
"Nieuwstadt, F. T., Mason, P. J., Moeng, C. H., & Schumann, U. (1993). Large-eddy simulation of the convective boundary layer: A comparison of four computer codes. In Turbulent shear flows 8 (pp. 343-367). Springer, Berlin, Heidelberg."

File: Cases.pyx [hunk 3]
```diff
@@ -70,6 +72,7 @@ cdef class CasesBase:
 
 
 cdef class Soares(CasesBase):
+# Case based on (Soares et al, 2004): An EDMF parameterization for dry and shallow cumulus convection
```
Comment (yairchn) [inline Cases.pyx]: maybe the full reference:
"Soares, P. M. M., Miranda, P. M. A., Siebesma, A. P., & Teixeira, J. (2004). An eddy‐diffusivity/mass‐flux parametrization for dry and shallow cumulus convection. Quarterly Journal of the Royal Meteorological Society: A journal of the atmospheric sciences, applied meteorology and physical oceanography, 130(604), 3365-3383."

File: Cases.pyx (hunk 2 above)
Comment (ilopezgp) [reply Cases.pyx]: Yes, I was just being lazy. Definitely better

**Pull request merged**